CONTRIBUTING.rst

```diff
@@ -0,0 +1,63 @@
+How to contribute
+=================
+
+Welcome to *coala*! The following guidelines will give you a short overview 
+over how we handle issues and PRs in this repository. *coala* is maintained
+by a growing community and newcomers are always welcome!
+
+We appreciate all valuable contributions - bug reports, documentation and
+design enhancements issues as well as pull Requests to make *coala* more
+convenient, functional and bug free.
+
+To begin contributing have a look at the open issues labeled with
+difficulty/newcomer - `newcomer issues <https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer>`__.
+
+Follow the instructions `here <http://coala.readthedocs.org/en/latest/Getting_Involved/README.html>`__ and get involved to help us make a better
+software.
+
+Filing Issues
+-------------
+
+When filing an issue, please use this template:
+
+::
+
+    ### Overview Description
+
+    # Steps to Reproduce
+
+    1.
+    2.
+    3.
+
+    # Actual Results
+
+    # Expected Results
+
+    # Reproducibility
+
+    # Additional Information:
+
+PR Merge Criteria
+-----------------
+
+For a PR to be merged, the following statements must hold true:
+
+-  All CI services pass. (CircleCI, AppVeyor, Scrutinizer, CodeCov)
+-  All commits must have been reviewed and approved by GitMate and a
+   *coala*
+   maintainer who is not the author of the PR.
```

Comment:
why is this on two lines? this text should be on the line above probably, this formatting looks weird.